docs/home/changelog/hotfixes-2-15.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Changelog Version 2.15
+summary: This article lists the fixed and known issues in the patch versions as well as additional new features that were added in each patch version.
+authors:
+    - Jason Novich
+date: 2024-Feb-25
+---
+
+The following is a list of the known and fixed issues for Run:ai V2.15.
+
+## Version 2.15.9 - February 5, 2024
+
+### Fixed issues
+
+| Internal ID | Description |
+|--|--|
+| RUN-15296 | Fixed an issue where the `resources` field was deprecated when creating a project. Now use `nodePoolResources. |
+
+## Version 2.15.4 - January 5, 2024
+
+### Fixed issues
+
+| Internal ID | Description |
+|--|--|
+| RUN-15026 | Fixed an issue where building workloads to send to the cluster were not built inside volumes for backwards compatibility. |
+| RUN-14907 | Fixed an issue where after an upgrade, the Analytics Dashboard is missing the time ranges from before the upgrade.|
+| RUN-14903 | Fixed an issue where the project status is audited creating long logs. |
```

Comment:
I would remove it. Does not seem customer facing + KCL issues are mapped usually by our team